test-end-to-end/android-maestro/e2e-tests/003_007_all_user_able_to_open_text_block.yaml

```diff
@@ -0,0 +1,82 @@
+appId: com.toughra.ustadmobile
+env:
+  TESTRESULTSDIR: "${TESTRESULTSDIR}/003_007_all_user_able_to_open_text_block"
+  TESTNAME: "003_007_all_user_able_to_open_text_block"
+  TEXTTITLE: "PRIMARY COLOURS"
+---
+- clearState: com.toughra.ustadmobile
+- runFlow:
+    file: "../common/startserver.yaml"
+    env:
+      CONTROLSERVER: ${CONTROLSERVER}
+      TESTSERIAL: ${TESTSERIAL}
+      TESTNAME: ${TESTNAME}
+- runFlow: "../common/clear_state_and_get_started.yaml"
+- runFlow: "../common/select_site.yaml"
+- runFlow:
+    file: "../common/user_login.yaml"
+- runFlow:
+    file: "../common/add_course.yaml"
+    env:
+      TESTCOURSENAME: "Text Block Test"
+- runFlow:
+    file: "../common/add_new_teacher_inside_course.yaml"
+    env:
+      TESTCOURSENAME: "Text Block Test"
+      TEACHERFIRSTNAME: "Julie"
+      TEACHERLASTNAME: "Susan"
+      GENDER: "Female"
+      PASSWORD: "test"
+- runFlow:
+    file: "../common/add_new_student_inside_course.yaml"
+    env:
+      TESTCOURSENAME: "Text Block Test"
+      STUDENTFIRSTNAME: "Muhammed"
+      STUDENTLASTNAME: "Khan"
+      GENDER: "Male"
+      PASSWORD: "test"
+- back
+- back
+- tapOn: "OVERVIEW"
+- runFlow:
+    file: "../common/add_demo_block_structure.yaml"
+    env:
+      MODULETITLE: "SEMESTER 1"
+      TEXTTITLE: "PRIMARY COLOURS"
+      TEXT: "RED,BLUE,GREEN"
+- tapOn: "PRIMARY COLOURS"
+- extendedWaitUntil:
+    visible: ${TEXTTITLE}
+    timeout: 1000
+- takeScreenshot: "${TESTRESULTSDIR}/text_block_admin_view"
+- runFlow: "../common/clear_state_and_get_started.yaml"
+- runFlow: "../common/select_site.yaml"
+- runFlow:
+    file: "../common/user_login.yaml"
+    env:
+      USERNAME: "Muhammed"
+      PASSWORD: "test"
+- tapOn: "courses"
+- tapOn: "Text Block Test"
+#--- open text block by student---#
+- tapOn: "PRIMARY COLOURS"
+- extendedWaitUntil:
+    visible: ${TEXTTITLE}
+    timeout: 1000
+- takeScreenshot: "${TESTRESULTSDIR}/text_block_student_view"
```

Comment:
Needs an assertion that the text itself is visible...

Reply:
Done Mike